aep/general/0135/long_running_delete.oas.yaml

```diff
@@ -0,0 +1,63 @@
+openapi: 3.0.3
+info:
+  title: Library
+  version: 1.0.0
+paths:
+  /publishers/{publisherId}/books/{bookId}:
+    parameters:
+    - $ref: "#/components/parameters/PublisherId"
+    - $ref: "#/components/parameters/BookId"
+    delete:
+      operationId: deleteBook
+      description: Delete a single book.
+      responses:
+        '202':
+          description: Accepted
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DeleteBookOperation'
+components:
+  parameters:
+    PublisherId:
+      name: publisherId
+      in: path
+      description: The id of the book publisher.
+      required: true
+      schema:
+        type: string
+    BookId:
+      name: bookId
+      in: path
+      description: The id of the book.
+      required: true
+      schema:
+        type: string
+  schemas:
+    DeleteBookOperation:
+      description: The status of the deleteBook operation.
+      properties:
+        response:
+          type: object
+          description: Empty object.
+        metadata:
+          type: object
+          properties:
+            startTime:
+              type: string
+              format: date-time
+              description: The time the operation started.
+            progressPercent:
+              type: integer
+              format: int32
+              description: The current progress, expressed as an integer.
+            state:
```

Comment:
the OAS annotations are very cool! cc @earth2marsh too if he wants to take a look.

For common values like state: should we be just referencing some stock schemas / values? it might get difficult to maintain all of these oas files with some duplicated content.

Reply:
Let's defer this to the point where we have a bulk of OAS examples merged so we can made an informed judgement on what should go into common schemas.